core/pom.xml

```diff
@@ -138,6 +138,13 @@
       <artifactId>json-simple</artifactId>
       <version>1.1</version>
     </dependency>
+    
+    <!--  org.json  -->
+    <dependency>
+        <groupId>org.json</groupId>
```

Comment:
please remove this dependency, or add some comments on why it is needed. I expect we already have json libraries in this project.